File: ExtendedDataGridView Control/[C#]-ExtendedDataGridView Control/C#/ExtendedDatagridViewCS/Printing/xC.cs
namespace ExtendedDataGridView
{
    public class xC
	{

		private int _X;

		private int _Width;

		public xC()
		{
		}

		public xC(int x, int w)
		{
			this.X = x;
			this.Width = w;
		}

		public int X {
			get { return _X; }
			set { _X = value; }
		}

		public int Width {
			get { return _Width; }
			set { _Width = value; }
		}

	}
}

File: ExtendedDataGridView Control/[VB]-ExtendedDataGridView Control/VB/ExtendedDataGridView/ExtendedDataGridView/Printing/xC.vb
Public Class xC

    Private _X As Integer
    Private _Width As Integer

    Public Sub New()

    End Sub

    Public Sub New(x As Integer, w As Integer)
        Me.X = x
        Me.Width = w
    End Sub

    Public Property X As Integer
        Get
            Return _X
        End Get
        Set(value As Integer)
            _X = value
        End Set
    End Property

    Public Property Width As Integer
        Get
            Return _Width
        End Get
        Set(value As Integer)
            _Width = value
        End Set
    End Property

End Class
